deployments/qat_plugin/overlays/sriov_numvfs/sriov_numvfs_init.yaml

```diff
@@ -20,6 +20,7 @@ spec:
           done
         securityContext:
           readOnlyRootFilesystem: true
+          allowPrivilegeEscalation: false
```

Comment:
this one triggers:
`cannot set `allowPrivilegeEscalation` to false and `privileged` to true\n\nerror:\nexit status 1",` in e2e-qat

Reply:
yep, removed